README.md

```diff
@@ -111,30 +125,7 @@ Analyzing 175 revisions...
  94 (  4; 175)  hartwork
 ```
 
-
-# Installation
-
-If your Linux distribution of choice does not come with a package for **svneverever** yet,
-you can install **svneverever** using pip from PyPI
-
-```console
-# pip install svneverever
-```
-
-or from a Git clone:
-
-```console
-# git clone https://github.com/hartwork/svneverever.git
-# cd svneverever
-# python3 setup.py install --user
-# echo 'export PATH="${HOME}/.local/bin:${PATH}"' >> ~/.bashrc
-```
-
-Besides Python 3.5/3.6, **svneverever**'s only dependency is
-[pysvn](https://pysvn.sourceforge.io/).
-
-
-# Usage
+# Help #
```

Comment:
I get the idea but this headline seems a bit weird to me. How about "`--help` output" instead or so?